vault/tests/vault_integration_tests.rs

```diff
@@ -126,17 +131,25 @@ async fn test_replace_succeeds() {
 
         let btc_rpcs = vec![(new_vault_id.clone(), btc_rpc.clone())].into_iter().collect();
         let new_btc_rpc_master_wallet = btc_rpc.clone();
-        let _vault_id_manager =
-            VaultIdManager::from_map(new_vault_provider.clone(), new_btc_rpc_master_wallet, btc_rpcs);
+        let _vault_id_manager = VaultIdManager::from_map(
+            new_vault_provider.clone(),
+            new_btc_rpc_master_wallet,
+            btc_rpcs,
+            "test_replace_succeeds1",
+        );
```

Comment:
Doesn't look like we need this, or is the construction required?

Reply:
It's just to ensure that tests don't interfere with each other by reading eath other's databases. We could use something like tempdir but I think for now this is fine